build.gradle

```diff
@@ -32,53 +47,60 @@ dependencies {
 
     modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
 
-    include modImplementation("dev.isxander.yacl:yet-another-config-lib-fabric:${project.yacl_version}") {
```

Comment:
Please revert this; I'm aware it's not recommended by the author, but my time modding and giving user support tells me it's in fact not likely the user will already have YACL on their mod folder.